### PR TITLE
use k8s.io/kubelet

### PR DESCRIPTION
**Pull request by yseto**
we were copying it to deal with an event like that in #55.

According to the https://github.com/kubernetes/kubernetes/tree/0fe90ac48b5c506ee60c2751b1dbcc4e2e053490/staging
, the policy has been changed as follows

> This directory is the staging area for packages that have been split to their own repository. The content here will be periodically published to respective top-level k8s.io repositories.

Therefore, switch the reference and delete the copied contents.

### CODE REVIEW

File: platform/kubernetes/kubelet/client.go [hunk 1]
```diff
@@ -10,7 +10,8 @@ import (
 	"path"
 	"regexp"
 
-	kubeletTypes "github.com/mackerelio/mackerel-container-agent/internal/k8s-apis/stats/v1alpha1"
+	kubeletTypes "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
+
```
Comment (susisu) [inline platform/kubernetes/kubelet/client.go:14]: [nit] Maybe this and the next file's empty lines are redundant?

**Comment by susisu** [comment]
This task should also be updated.
https://github.com/mackerelio/mackerel-container-agent/blob/ae380315d269bb20f8662c0d8b3cfe03371f3280/Makefile#L48-L52

File: platform/kubernetes/kubelet/client.go (hunk 1 above)
Comment (yseto) [reply platform/kubernetes/kubelet/client.go]: fixed at 1a865cd54b3a4eaf808af297924058c82843066a, f1fcafdc5e2112b1334be63ba083dfc0b2dc691f

**Review by susisu — APPROVED** [review]
LGTM~

**Comment by yseto** [comment]
Thanks for your review!

**Pull request merged**